osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -78,7 +87,20 @@ protected override void LoadComplete()
                     : "press for menu";
             }, true);
 
-            text.FadeInFromZero(500, Easing.OutQuint).Delay(1500).FadeOut(500, Easing.OutQuint);
+            touchActive = sessionStatics.GetBindable<bool>(Static.TouchInputActive);
+
+            if (touchActive.Value)
```

Comment:
Is this intentionally checked only once ever?

It's a bit weird for me to first take out a config bindable to then only check it once...

Reply:
It's used in `Update`  as well, to get the minimum alpha level. This one is just to decide how to animate, and there's not much point in checking beyond the initial load.